content/articles/using-ecs-to-deploy-docker-app-to-awws/index.md

```diff
@@ -0,0 +1,232 @@
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Docker | AWS Index</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
+    <div class="jumbotron text-center">
+        <h2>FLASK APP DEPLOYED TO AWS USING DOCKER</h4>
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Illum officia asperiores impedit possimus quas 
+        officiis expedita velit, at architecto iusto natus modi quaerat, nulla laboriosam atque odio amet debitis 
+        et voluptas ea! Pariatur autem tempora placeat saepe doloribus minus ab maxime excepturi neque illo. Ratione,
+        sapiente magnam? Perspiciatis, molestias nihil.</p>
+    </div>
+    </div>   
+    </div> 
+</body>
+</html>
+```
+Now, let us run the flask app to see if it works. In the terminal, execute the command below:
+```bash
+set FLASK_APP='app.py'
+flask run -p 5000
+```
+You should have the information below in the terminal:
+```bash
+* Environment: production
+WARNING: This is a development server. Do not use it in a production deployment.
+Use a production WSGI server instead.
+* Debug mode: off
+* Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+```
+Now head over to the browser on  `http://127.0.0.1:5000/` to see if the application is running.
+![App running locally](/engineering-education/using-ECS-to-deploy-a-docker-app-to-AWS/app-running.png)
+
+### Dockerizing the flask app
+Now that our application is up and running, we need to dockerize it. Dockerization is packaging the application and its environmental libraries and dependencies into one so that the app can run anywhere without having to perform new environmental configurations.
+
+First, we need to create a `dockerfile`. The docker file is used by the Docker engine to create a new docker image of the application container. It sets up an environment needed to run the application.
+Create a file called `Dockerfile` and add the snippets below:
+```py
+# For more information, please refer to https://aka.ms/vscode-docker-python
+FROM python:3.8
+
+# working directory
+WORKDIR /user/src/app
+
+# copy all files to the container
+COPY .  .
+
+# Install pip requirements
+RUN python -m pip install --no-cache-dir -r requirements.txt
+
+# port number to expose
+EXPOSE 5000
+
+# run the command
+CMD ["python", "./app.py"]
+```
+### Running the container
+In this phase, we will combine all the files and run the docker container.
+Startup the docker desktop the execute the command below:
+```bash
+docker build -t app .
+```
+You should have the docker execute each line of the `Dockerfile` as shown below:
+![Docker Build](/engineering-education/using-ECS-to-deploy-a-docker-app-to-AWS/build-app.png)
```

Comment:
Change `using-ECS-to-deploy-a-docker-app-to-AWS/` to `using-ecs-to-deploy-docker-app-to-aws/`.

Also, you have typed `aws` as `awws` in the directory name. Please rectify that.

Reply:
Do this for all the images. And, make sure to rename the directory.